components/HomepageSubnav.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from 'next/link';
+import styles from '/styles/HomepageSubnav.module.scss';
+
+type HomepageSubnavProps = {
+    title: string;
+    jobs: number;
+    children: React.ReactNode;
+};
+
+export const HomepageSubnav = ({
+    title,
+    jobs,
+    children,
+}: HomepageSubnavProps) => (
+    <div className={styles.container}>
+        <ul className={styles.navigation}>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#benefits">
+                    Benefits
+                </a>
+            </li>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#lifeAsATorchboxer">
+                    Life as a Torchboxer
+                </a>
+            </li>
+            <li className={styles.jobNavItem}>
+                <Link href="/jobs">
+                    <a className={styles.jobsLink}>
+                        Jobs
+                        {jobs !== undefined && jobs > 0 ? (
+                            <div className={styles.jobsBadge}>{jobs}</div>
+                        ) : null}
+                    </a>
+                </Link>
+            </li>
+        </ul>
+        <img
```

Comment:
Lint warning
```Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element.```

We'd have to use something like the snippets below
```js
import Image from 'next/image'
...
        <div className={styles.image}>
            <Image
                src="/images/arrow-loop-right.svg"
                height={123}
                width={95}
                objectFit='contain'
            />
        </div>
        <div className={styles.image} style={{ width: '95px', height: '123px', position: 'relative' }}>
            <Image
                src="/images/arrow-loop-right.svg"
                alt="Close Nav Bar"
                layout='fill'
                objectFit='contain'
            />
        </div>
```

Reply:
Looks like your work from the social media images PR should help with this https://github.com/torchbox/careers/pull/22/

Reply:
I don't think we should lazyload this SVG if we're loading it as part of the page render, I think I will create a custom SVG component instead as you've suggested above